shared-data/python/opentrons_shared_data/gripper/dev_types.py

```diff
@@ -0,0 +1,79 @@
+"""Types for gripper config that require typing_extensions.
+
+This module should only be imported if typing.TYPE_CHECKING is True.
+"""
+from typing import Dict, List, Union, NewType
+
+from typing_extensions import Literal, TypedDict
+
+
+GripperName = Literal["gripper"]
+
+GripperModel = NewType("GripperModel", str)
+
+DisplayCategory = Literal["GEN1"]
+
+ConfigUnit = Union[
+    Literal["amps"],
+    Literal["hertz"],  # PWM frequency
+    Literal["percentage"],  # Duty cycle
+    Literal["volt"],  # Reference voltage
+]
```

Comment:
`Union[Literal[...], Literal[...], ...]` is redundant

```suggestion
ConfigUnit = Literal[
    "amps",
    "hertz",  # PWM frequency
    "percentage",  # Duty cycle
    "volt",  # Reference voltage
]
```